src/array.jl

```diff
@@ -729,6 +729,10 @@ function in(x::CategoricalValue, y::CategoricalArray{T, N, R}) where {T, N, R}
     end
 end
 
+collect(x::CategoricalArray{T}) where {T} = convert(Array{T}, x)
+collect(x::Array{CategoricalValue{T,R}}) where {T,R} = convert(Array{T}, x)
```

Comment:
What's the rationale for supporting this method and the one below? Shouldn't we prevent people from ending with this kind of array in the first place?

Reply:
You're right, we should rather make it easy to to the right thing and not encourage working with such arrays. I'll remove them

Reply:
The rationale btw was to be able to work with `collect(Missings.replace(...))` as suggested by the documentation for `Missings`. But `collect(collect(Missings.replace(...)))` isn't exactly ideal ...